test/testUtils/factories.js

```diff
@@ -212,6 +212,21 @@ export const setUpScheduledCall = async (args = {}) => {
 };
 
 export const setupVisit = async (args = {}) => {
+  const visit = {
+    patientName: "Patient Name",
+    contactEmail: "contact@example.com",
+    contactName: "Contact Name",
+    callTime: new Date("2020-06-01 13:00"),
+    contactNumber: "01234578912",
+    ...args,
+  };
+  return await container.getCreateScheduledCallGateway(container)(
+    visit,
+    args.wardId
+  );
+};
+
+export const setupVisitPostgres = async (args = {}) => {
```

Comment:
Probably the best way to do this so that it doesn't affect all the old postgres tests is not refactor setupVisit(which uses the old pg GW) but use setupScheduledCall which is now merged into master . 🙂